package.json

```diff
@@ -61,7 +61,11 @@
 	},
 	"publishConfig": {
 		"ignore": [
-			".github/workflows"
+			".github/workflows",
+			"example",
```

Comment:
while this is technically a breaking change, i doubt anyone’s relying on example files being present.